src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -0,0 +1,257 @@
+﻿#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Windows.UI.Notifications
+{
+	public partial class ToastNotifier
+	{
+		private string _channelID = "PlatformUno"; 
+		private Android.App.NotificationManager _notificationManager;
+
+		public ToastNotifier()
+		{
+			_notificationManager = Android.App.NotificationManager.FromContext(Android.App.Application.Context);
+
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.O)
+			{	// none of this strings would be visible for user
+				var channel = new Android.App.NotificationChannel(_channelID, "PlatformUnoChannel", Android.App.NotificationImportance.Default); ; // deprecated: Android.App.NotificationManager.ImportanceDefault
+				_notificationManager.CreateNotificationChannel(channel);
+			}
+
+		}
+
+		private string toastText2String(string toasttext)

```

Comment:
```suggestion
		private string ConvertToastTextToString(string toastText)

```

Please use camel casing, pascal casing where appropriate. Use the Visual studio cleanup tool to do this.